src/BulkDocs/BulkDocs.php

```diff
@@ -39,17 +41,20 @@ class BulkDocs implements BulkDocsInterface {
   static public function createInstance(ContainerInterface $container, WorkspaceManagerInterface $workspace_manager, WorkspaceInterface $workspace) {
     return new static(
       $workspace_manager,
-      $workspace
+      $workspace,
+      $container->get('relaxed.stub_entity_processor')
     );
   }
 
   /**
    * @param \Drupal\multiversion\Workspace\WorkspaceManagerInterface $workspace_manager
    * @param \Drupal\multiversion\Entity\WorkspaceInterface $workspace
+   * @param \Drupal\relaxed\StubEntityProcessor\StubEntityProcessorInterface $stub_entity_processor
    */
-  public function __construct(WorkspaceManagerInterface $workspace_manager, WorkspaceInterface $workspace) {
+  public function __construct(WorkspaceManagerInterface $workspace_manager, WorkspaceInterface $workspace, StubEntityProcessorInterface $stub_entity_processor) {
     $this->workspaceManager = $workspace_manager;
     $this->workspace = $workspace;
+    $this->stub_entity_processor = $stub_entity_processor;
```

Comment:
Object properties should always use camel case for their name. So this should be `$this->stubEntityProcessor`.